src/config.ts

```diff
@@ -69,6 +69,53 @@ const conf = {
     headers: {
       'X-Auth-Type-Kiali-UI': '1'
     }
+  },
+  /** API configuration */
+  api: {
+    urls: {
+      apps: (namespace: String) => `api/namespaces/${namespace}/apps`,
```

Comment:
Can we use typescript `string` instead of `String`?

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html